packages/dcos-integration-test/extra/test_ucr.py

```diff
@@ -0,0 +1,208 @@
+
+
+def test_if_ucr_pods_can_be_deployed_with_image_entrypoint(dcos_api_session):
+    """Marathon pods inside ucr deployment integration test.
+
+    This test verifies that a Marathon pods can be deployed.
+    """
+    test_uuid = uuid.uuid4().hex
+    pod_definition = {
+        'id': '/integration-test-pods-{}'.format(test_uuid),
+        'scaling': {'kind': 'fixed', 'instances': 1},
+        'containers': [
+            {
+                'name': 'image_entrypoint',
+                'resources': {
+                    'cpus': 0.1,
+                    'mem': 32
+                },
+                'image': {
+                    'kind': 'DOCKER',
+                    'id': 'mesosphere/inky'
+                }
+            }
+        ],
+        'networks': [{'mode': 'host'}]
+    }
+    dcos_api_session.marathon.deploy_pod_and_cleanup(pod_definition)
+
+
+def test_if_ucr_pods_can_be_deployed_with_scratch_image(dcos_api_session):
+    """Marathon pods inside ucr deployment integration test.
+
+    This test verifies that a Marathon pods can be deployed.
+    """
+    test_uuid = uuid.uuid4().hex
+    pod_definition = {
+        'id': '/integration-test-pods-{}'.format(test_uuid),
+        'scaling': {'kind': 'fixed', 'instances': 1},
+        'containers': [
+            {
+                'name': 'image_entrypoint',
+                'resources': {
+                    'cpus': 0.1,
+                    'mem': 32
+                },
+                'image': {
+                    'kind': 'DOCKER',
+                    'id': 'hello-world'
+                }
+            }
+        ],
+        'networks': [{'mode': 'host'}]
+    }
+    dcos_api_session.marathon.deploy_pod_and_cleanup(pod_definition)
+
+
+def test_if_ucr_pods_can_be_deployed_with_image_whiteout(dcos_api_session):
+    """Marathon pods inside ucr deployment integration test.
+
+    This test verifies that a Marathon pods can be deployed.
+    """
+    test_uuid = uuid.uuid4().hex
+    pod_definition = {
+        'id': '/integration-test-pods-{}'.format(test_uuid),
+        'scaling': {'kind': 'fixed', 'instances': 1},
+        'containers': [
+            {
+                'name': 'image_entrypoint',
+                'resources': {
+                    'cpus': 0.1,
+                    'mem': 32
+                },
+                'image': {
+                    'kind': 'DOCKER',
+                    'id': 'gilbertsong/whiteout'
+                },
+                'exec': {
+                    'command': {
+                        'shell': 'test ! -f /dir1/file1 && test ! -f /dir1/dir2/file2 && test -f /dir1/dir2/file3'
+                    }
+                }
+            }
+        ],
+        'networks': [{'mode': 'host'}]
+    }
+    dcos_api_session.marathon.deploy_pod_and_cleanup(pod_definition)
+
+
+def test_if_ucr_pods_can_be_deployed_with_image_digest(dcos_api_session):
+    """Marathon pods inside ucr deployment integration test.
```

Comment:
Following the current existing pattern from `test_if_marathon_pods_can_be_deployed_with_mesos_containerizer()`. Also think this comment is not intuitive.

Do you want me to remove all comments like this in this file?